xml/ay_bigfile.xml

```diff
@@ -2363,10 +2363,8 @@ openssl x509 -noout -fingerprint -sha256
         </entry>
         <entry>
          <para>
-          If set to <literal>true</literal>, booting from Btrfs snapshots
-          will be enabled.
+          Obsolete and no longer used. It is auto detected nowadays.
```

Comment:
Thanks for the PR! \o/ For this, I suggest 'Obsolete and no longer used. Booting from Btrfs snapshots is automatically enabled since [SLES version].' or something like that. It is important to say which SLES version.